src/LinuxTracing/PerfEventOpen.cpp

```diff
@@ -92,6 +92,12 @@ int callchain_sample_event_open(uint64_t period_ns, pid_t pid, int32_t cpu) {
   pe.sample_max_stack = 127;
   pe.exclude_callchain_kernel = true;
 
+  // Also capture a small part of the stack and the registers to allow patching the callers of
+  // leaf functions. This is done by unwinding the first frame using DWARF.
```

Comment:
The first two? Or is the first one enough and then we already have all the information to determine the caller?